### ISSUE TITLE
Error "The plugin "react-display-name" didn't export a Plugin instance"

### ISSUE THREAD

**Issue by hackingbeauty** [insin/nwb#54]
Hello,

When I try to run:

```
nwb serve
```

I get the error "TypeError: The plugin "react-display-name" didn't export a Plugin instance".

Can somebody help me with this?

**Comment by insin** [comment]
Is the version of `nwb` you're using installed globally or locally?

Which versions of Node.js, npm and nwb are you using?
```
node -v
npm -v
nwb -v (if installed globally)
```

Which modules are installed in your project?
```
npm ls --depth=0
```

Do you have any other potential sources of Babel config in your project, such as a `.babelrc`?

**Comment by hackingbeauty** [comment]
Hey Jonny, thanks for the tip and sorry to waste your time.  I had a .babelrc file one directory above my nwb project.  There was no issue with nwb.  Thank you for helping me out.

**Comment by insin** [comment]
Not at all, and thanks for raising it - I think nwb could have prevented that being an error using Babel's hidden `breakConfig` option, plus I needed to write something like the above post to be added to the CONTRIBUTING doc, so that's 2 useful outcomes from this issue :+1: :+1:

**Comment by insin** [comment]
Closing this as the underlying issue will be fixed in #63 and CONTRIBUTING.md has been updated with a self-debugging checklist.
